shared/version/api.go

```diff
@@ -388,7 +388,6 @@ var APIExtensions = []string{
 	"instance_move_config",
 	"ovn_ssl_config",
 	"init_preseed_storage_volumes",
-	"metrics_instances_count",
```

Comment:
Can you use a git revert of the original combined api extension commit for this please

Reply:
fixed